client/app/dedicatedCloud/dedicatedCloud.controller.js

```diff
@@ -37,6 +39,30 @@ angular.module("App").controller("DedicatedCloudCtrl", [
 
         $scope.dedicatedCloud = {};
 
+        function showNotificationIfRequired (notification) {
```

Comment:
You're mixing ways of creating functions here, not sure this is a good idea (other functions are declared this way:
`$scope.functionName = function (arguments)`

Reply:
OK for the two styles, but maybe I should write $scope.function for public ones, and keep function MyFunction for the private ones ?